### ISSUE TITLE
Add graphs visualization with GraphViz

### ISSUE THREAD

**Issue by ileasile** [Kotlin/kotlin-jupyter#219]
GraphViz-java library provides a cool Kotlin API for the rendering of the graphs, but even more importantly, it provides a way to render these graphs and `.dot` files. Make it possible to use this library in the kernel: either provide this functionality in lib-ext or make a separate API wrapper or [contribute to the library itself](https://github.com/nidi3/graphviz-java) or make a JSON wrapper inside the kernel.

**Comment by altavir** [comment]
There is very nice [kaliningraph](https://github.com/breandan/kaliningraph) library by @breandan and you already have [integration with it](https://github.com/Kotlin/kotlin-jupyter/blob/master/libraries/kaliningraph.json).

**Comment by ileasile** [comment]
Yes, I know, but graphviz seems to be a bit more stable solution at the moment. However, I see that kaliningraph uses graphviz under the hood

**Comment by breandan** [comment]
[Kaliningraph](https://github.com/breandan/kaliningraph) uses [Kraphviz](https://github.com/nidi3/graphviz-java/blob/master/graphviz-kotlin/src/main/kotlin/guru/nidi/graphviz/Kraphviz.kt), which uses [graphviz-java](https://github.com/nidi3/graphviz-java), which uses [viz.js](https://github.com/mdaines/viz.js), which uses [Graphviz](http://www.graphviz.org/) recompiled for WASM. As Kaliningraph is currently a research prototype I would not recommend using it for anything important. Depending on what you need to do, it might be better to use viz.js or another graph library such as [G6](https://github.com/antvis/G6), [neoviz.js](https://github.com/neo4j-contrib/neovis.js) or another [graph visualization library](https://github.com/topics/graph-visualization). You could also reach out to @alshan who might know more about writing a ggplot-style API or wrapper.
